packages/php-wasm/web/src/lib/tcp-over-fetch-websocket.spec.ts

```diff
@@ -38,6 +89,29 @@ describe('TCPOverFetchWebsocket', () => {
 			res.send('OK');
 		});
 
+		app.get('/gzipped', (req, res) => {
+			const gzip = zlib.createGzip();
+			gzip.write(pygmalion);
+			gzip.end();
+
+			const gzippedChunks: Uint8Array[] = [];
+			gzip.on('data', (chunk) => {
+				gzippedChunks.push(chunk);
+			});
```

Comment:
Is it possible that there is only ever one chunk, and if so, would it matter?

Reply:
It is possible and it seems fine – it would push only one chunk. Or do you have some concerns about this?

Reply:
> It is possible and it seems fine – it would push only one chunk. Or do you have some concerns about this?

Intuitively, I expected testing to involve a response made of multiple chunks and realized that that wasn't guaranteed here. If we don't need to test that here, then it doesn't matter. I still need to finish reading the TLS 1.2 PR and don't have an opinion yet. :)

Reply:
Ah, I see! In this test, it only matters that we get a Content Length that represents the compressed data and is inconsistent with the uncompressed data

Reply:
Thanks for explaining!